mention-storer/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>de.brandwatch.mini-analytics</groupId>
```

Comment:
It would be nice if your pom files could have the same ordering of things (especially the high level tags). You can choose to do it yourself or utilise some configuration of https://github.com/Ekryd/sortpom.